commit ea4397ee3a3b7b324eb1ef8c90c877ef9db226df
Author: Jose Dapena Paz <jdapena@igalia.com>
Date:   Wed Dec 20 19:16:36 2023 +0000

    IWYU: missing include for vector usage in crypto/hkdf.h
    
    Bug: 957519
    Change-Id: Ia96e2d0ee13b2f682aaae8c05381a7413815bd43
    Reviewed-on: https://chromium-review.googlesource.com/c/chromium/src/+/5138462
    Reviewed-by: Adam Langley <agl@chromium.org>
    Commit-Queue: Adam Langley <agl@chromium.org>
    Cr-Commit-Position: refs/heads/main@{#1239774}

diff --git a/crypto/hkdf.h b/crypto/hkdf.h
index c9aeabe2b6f8f..8fef8c6d257d5 100644
--- a/crypto/hkdf.h
+++ b/crypto/hkdf.h
@@ -9,6 +9,7 @@
 
 #include <string>
 #include <string_view>
+#include <vector>
 
 #include "base/containers/span.h"
 #include "crypto/crypto_export.h"
